.github/workflows/ci.yml

```
@@ -2,32 +2,86 @@
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
+  # Fast preliminary checks - fail fast strategy
+  check:
+    name: Check & Lint
+    runs-on: ubuntu-latest
+    outputs:
+      cache-key: ${{ steps.cache-key.outputs.key }}
```

actionlint (1.7.7)

property "cache-key" is not defined in object type {}

(expression)